### PR TITLE
Add script

### PR DESCRIPTION
**Pull request by shiki-tak**
## Description

closes: 

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by ulbqb — CHANGES_REQUESTED** [review]
Please indicate the environment in which L1/simd works.

**Comment by shiki-tak** [comment]
> Please indicate the environment in which L1/simd works.

The following commit.
https://github.com/Finschia/finschia-sdk/commit/25043656ed1a6a4947b2c5c3804d724bbcfc43ec

**Comment by ulbqb** [comment]
@shiki-tak 
Sorry for lack of explanation. Please rename this to simd-linux-arm64 or simd-darwin-arm64.

**Review by ulbqb — APPROVED** [review]

File: demo/test_sequencer.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+./prepare_rollup.sh
+
+cd ..
+
+# Build & rename
+make build && cp -r build/simd $GOPATH/bin/$L2BINARYNAME
+${L2BINARYNAME} version
+
+# Init sequencer
+zsh init_node.sh $L2_CHAIN_ID
```
Comment (ulbqb) [inline demo/test_sequencer.sh:36]: The first argument of init_node.sh is the number of nodes.

File: demo/test_sequencer.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+L2BINARYNAME=rollupd
+ROLLUP_NAME=test-rollup
+GOPATH=$HOME/go
+L1_CHAIN_ID=sim
+L2_CHAIN_ID=sim2
+L1_KEYRING_DIR=~/.simapp
+L2_KEYRING_DIR=~/.l2simapp
```
Comment (ulbqb) [inline demo/test_sequencer.sh:9]: Where this directory is made?

Comment (shiki-tak) [reply demo/test_sequencer.sh:9]: here
https://github.com/Finschia/finschia-sdk/blob/for-ramus-sdk/init_node.sh#L75

**Review by Mdaiki0730 — APPROVED** [review]

**Review by ulbqb — APPROVED** [review]

**Pull request merged**